Sources/ComposableArchitecture/Documentation.docc/Articles/TreeBasedNavigation.md

```diff
@@ -627,19 +627,15 @@ as an enum instead of multiple optionals. In order to assert on state changes wh
 state you must be able to extract the associated state from the enum, make a mutation, and then
 embed the new state back into the enum.
 
-The library provides a tool to perform these steps in a single step, and it is called `XCTModify`:
+The library provides a tool to perform these steps in a single step. It's the
+``PresentationState/subscript(case:)-7uqte`` defined on ``PresentationState`` which allows you to
+modify the data inside a case of the destination enum:
```

Comment:
This was unrelated to this PR, but I just realized we should probably be pushing people towards the `[case:]` modify subscript instead of `XCTModify` by default. `XCTModify` is really only handy if you want to perform a bunch of mutations on the same case.